cookbook/lib/examples/gradient_bubbles.dart

```diff
@@ -86,12 +86,14 @@ class MessageBubble extends StatelessWidget {
             borderRadius: const BorderRadius.all(Radius.circular(16.0)),
             child: BubbleBackground(
               colors: [
-                message.isMine
-                    ? const Color(0xFF6C7689)
-                    : const Color(0xFF19B7FF),
-                message.isMine
-                    ? const Color(0xFF3A364B)
-                    : const Color(0xFF491CCB),
+                if (message.isMine) ...[
+                  const Color(0xFF6C7689),
+                  const Color(0xFF3A364B),
+                ],
+                if (!message.isMine) ...[
+                  const Color(0xFF19B7FF),
+                  const Color(0xFF491CCB),
+                ],
```

Comment:
Can't you use an `else` here? Or are you keeping them separate to illustrate something more clearly?

```suggestion
                if (message.isMine) ...[
                  const Color(0xFF6C7689),
                  const Color(0xFF3A364B),
                ] else ...[
                  const Color(0xFF19B7FF),
                  const Color(0xFF491CCB),
                ],
```

Reply:
For some reason I thought `else` wasn't available here. Today, I learned, and all that =)

Reply:
We still don't document collection control flow expressions well :( so a lot of people may not know.

https://github.com/dart-lang/site-www/issues/2093

Now I see the language tour doesn't even have an example with an `else`.